src/components/atoms/Input/index.tsx

```diff
@@ -36,7 +38,9 @@ const Input: React.FC<InputProps> = ({
         {
           'border-gray-600': errorIcon === false,
           'border-scrummyRed-500': errorIcon === true,
-          'mb-10': errorIcon,
+          'mb-[px]': errorIcon && errorPadding === 'small',
+          'mb-[54px]': errorIcon && errorPadding === 'normal',
```

Comment:
Why we're using `mb-[54px]` instead of `mb-14` which is 56px? Any particular reason?

Reply:
Thats my bad. I tried numbers like 13 and failed, so i decide to put in px, but i didin't see thats has  numbers like 14,16,20 and others...